reana_client/utils.py

```diff
@@ -191,15 +191,27 @@ def _validate_environment(reana_yaml):
         workflow_steps = reana_yaml["workflow"]["specification"]["steps"]
         _validate_serial_workflow_environment(workflow_steps)
     elif workflow_type == "yadage":
-        logging.warning(
-            "The environment validation is not implemented for Yadage workflows yet."
-        )
+        with open(reana_yaml["workflow"].get("file")) as f:
```

Comment:
@mvidalgarcia What do you think about loading the yadage steps in `load_reana_spec` somewhere or here it is fine?

Reply:
To prevent this maybe we can move [the environment validation](https://github.com/reanahub/reana-client/blob/d36de00b2b2f748de0bcc19cbb163c9ed2ec00f2/reana_client/utils.py#L151-L156) after [loading the workflow spec](https://github.com/reanahub/reana-client/blob/d36de00b2b2f748de0bcc19cbb163c9ed2ec00f2/reana_client/utils.py#L164), so we have access to the Yadage steps already. Would that make sense?

Reply:
@mvidalgarcia If we do that, we will have more options to explore the params in future by just passing `reana_yaml` in different methods.